README.md

```diff
@@ -0,0 +1,252 @@
+
+    $ mkdir ~/ACCESS-ESM
+    $ cd ~/ACCESS-ESM
+
+Then clone the most recent version of the ACCESS-ESM control directory:
+
+    $ git clone https://github.com/coecms/esm-pre-industrial pre-industrial
+    $ cd pre-industrial
+
+(Note: Currently we only have the pre-industrial model set up, other versions will follow later.)
+
+### Setting up the Master Configuration file.
+
+Open the `config.yaml` file with your preferred text editor.
+
+Let's have a closer look at the parts:
+
+    jobname: pre-industrial
+    queue: normal
+    walltime: 20:00:00
+
+These are settings for the PBS system. Name, walltime and queue to use.
+
+    # note: if laboratory is relative path, it is relative to /short/$PROJECT/$USER
+    laboratory: access-esm
+
+The location of the laboratory. At this point, payu can not expand shell environment variables (it's in our TO-DO), so as a work-around, if you use relative paths, it will be relative to your default short directory.
+
+In this default configuration, it will be in `/short/$PROJECT/$USER/access-esm`.
+But you can also hard-code the full path, if you want it somewhere different.
+
+    model: access
+
+The main model. This mainly tells PAYU which driver to use. PAYU knows that **access** is a coupled model, so it will look for separate configurations of the submodels, which is the next item of the configuration file:
+
+    submodels:
+        - name: atmosphere
+          model: um
+          ncpus: 192
+          exe: /short/public/access-esm/payu/bin/csiro/um_hg3.exe-20190129_15
+          input:
+            - /short/public/access-esm/payu/input/pre-industrial/atmosphere
+
+        - name: ocean
+          model: mom
+          ncpus: 84
+          exe: /short/public/access-esm/payu/bin/coe/fms_ACCESS-CM.x
+          input:
+            - /short/public/access-esm/payu/input/common/ocean
+            - /short/public/access-esm/payu/input/pre-industrial/ocean
+
+        - name: ice
+          model: cice
+          ncpus: 12
+          exe: /short/public/access-esm/payu/bin/csiro/cice4.1_access-mct-12p-20180108
+          input:
+            - /short/public/access-esm/payu/input/common/ice
+
+        - name: coupler
+          model: oasis
+          ncpus: 0
+          input:
+            - /short/public/access-esm/payu/input/common/coupler
+
+This is probably the meatiest part of the configuration, so let's look at it in more detail.
+
+Each submodel has
+- a **name**
+- the **model** to know which driver to use
+- the number of CPUs that this model should receive (**ncpus**)
+- the location of the executable to use (**exe**)
+- one or more locations for the **input** files.
+
+The **name** is more than a useful reminder of what the model is.
+PAYU expects this submodel's configuration files in a subdirectory with that name.
+
+    collate:
+       exe: /short/public/access-esm/payu/bin/mppnccombine
+       restart: true
+       mem: 4GB
+
+Ask Aidan
```

Comment:
These options specify the mom collate executable (for stitching the tiled outputs back together). This is submitted as a separate PBS job, and in this case needed more memory, as the default memory allocation was insufficient. The `restart: True` option means `payu` will also automatically collate the restart files from the previous run. This saves space as the original restart files are not compressed netCDF (it is an old version of the model) and there is a lot of duplication of dimension data. The collation process will enable netCDF compression on the collated files.